site/tool_configuration.md

```diff
@@ -1,10 +1,26 @@
 ## Tools Configuration
+ - [Tool Property File](#tool-properties)
  - [Model Filters](#model-filters)
  - [Target Environments](config/target_env.md)
  - [Type Definitions](#domain-type-definitions)
  - [Variable Injection](variable_injection.md)
  - [Custom Configuration](#custom-configuration)
 
+ ### Tool Properties
+ The WebLogic Deploy Tooling tools can be configured or tuned from the tool property file. This property file is installed as `<weblogic-deploy>/lib/tool.properties`. You may tune any of the properties in this file or override the file completely by providing your own tool.properties file. Reference the directory where the tool.properties resides using the  `CUSTOM_CONFIG_VARIABLE` Environment variable.
+ 
+ If a property is removed from the file, or a property value is incorrectly formatted, a `WARNING` message is logged and an internal default value used instead of the bad value.
+ 
+ | Property | Usage |
+ | -------- | ----- |
+ | connect.timeout | The number of milliseconds that WLST waits for the online connect command to complete. A value of 0 means the operation will not timeout. |        
```

Comment:
connect.timeout -> `connect.timeout` (property names in code font; globally)
connect -> `connect ` (commands in code font)